### PR TITLE
feat: support official mongo driver

### PR DESCRIPTION
**Pull request by egilsster**

### CODE REVIEW

**Comment by qlikossbuild** [comment]
Hi, and thanks for contributing!

Note: If you are part of Qlik R&D, you can ignore this process (but let us know in a comment below!)

Before we can handle your pull request we need you to sign our contributor license agreement.
This is a one-time thing and enables you to contribute without hassle to any of the qlik-oss repositories in the future.

And it only takes a few minutes! Here are some useful links to get you started:

* [Sign the CLA](https://github.com/qlik-oss/open-source/blob/master/sign-cla.md#sign-the-cla)
* [Rationale behind the CLA](https://github.com/qlik-oss/open-source/blob/master/cla-rationale.md)
* [Qlik R&D Open Source repository](https://github.com/qlik-oss/open-source#open-source-at-qlik-rd)
* [Official Slack team](http://branch.qlik.com/slack)

Having troubles? Feel free to reach out to us either here or on Slack!

**Comment by egilsster** [comment]
I am part of Qlik R&D

File: mongocursorpagination/mongo_find.go [hunk 1]
```diff
@@ -0,0 +1,349 @@
+// Package official mongo driver, eases the computation of pagination information
+// of a find mongo query by augmenting the base query with cursor information and
+// returning a cursor.
+package mongocursorpagination
```
Comment (EtienneDufresne) [inline mongocursorpagination/mongo_find.go:4]: Could you split `mgo` and `mongo` into two different packages? This is so a consumer does not have to vendor both drivers. So `mongocursorpagination/mgo` and `mongocursorpagination/mongo`

**Review by EtienneDufresne — COMMENTED** [review]
Could you update the README?

**Comment by egilsster** [comment]
Closing in favor of https://github.com/qlik-oss/mongocursorpagination/pull/10

**Pull request closed without merging**